Google.Api.Generator/RoslynUtils/RoslynConverters.cs

```diff
@@ -86,5 +87,36 @@ public static IEnumerable<StatementSyntax> ToStatements(object o)
 
         public static ArgumentListSyntax CreateArgList(params object[] args) =>
             ArgumentList(SeparatedList(args.SelectMany(ToArgs)));
+
+        public static SimpleNameSyntax ToSimpleName(object o, params TypeSyntax[] genericArgs)
+        {
+            if (o.GetType().IsEnum)
+            {
+                if (genericArgs.Any())
+                {
+                    throw new ArgumentException("Generic args must not be present for an enum");
+                }
+                return IdentifierName(o.ToString());
+            }
+            SyntaxToken name;
```

Comment:
Looking forward to switch expressions much? ;)

Reply:
Definitely.
I've been waiting for switch expressions for many C# versions! Scala has always had them, and they're lovely :)
I vote we switch to VS2019 and C# 8 previews *now*!